frontend/src/components/core/Text.tsx

```diff
@@ -8,14 +8,14 @@ export const ErrorText = styled(Text)`
    color: ${({ theme }) => theme.colors.red};
 `;
 
-export const MediumText = styled.p`
-  font-size: ${({ theme }) => theme.fontSize.xMediumLarge};
-`;
-
 export const LargeText = styled.h3`
   font-size: ${({ theme }) => theme.fontSize.xLarge};
 `;
 
+export const MediumText = styled.p`
+  font-size: ${({ theme }) => theme.fontSize.xMediumLarge};
+`;
+
```

Comment:
Good change 😉

Reply:
Have merge conflicts to thank for this haha